ibis/expr/datatypes.py

```diff
@@ -4,6 +4,7 @@
 import numbers
 import re
 
+from shapely import geometry
```

Comment:
The geospatial types should be moved into their own file, `geospatial.py` so that users who don't have `shapely` installed can still use the library.